src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+    currentTemp: 70,
+    backgroundColor: document.getElementById('buttom--left'),
+    landscapeImg: document.getElementById('buttom--right'),
+    cityInput: document.getElementById('city-input'),
+    location: null,
+    lat: null,
+    lon: null,
+    skyCondition: document.getElementById('sky--options'),
+    skyImg: document.getElementById('top')
+}
+// changes the temp on the page
+const changesTempDisplay = () => {
+    const tempControl = document.getElementById('temperature');
+    tempControl.textContent = `${state.currentTemp}`
+}
+// CHANGES TEMP
+const increaseTemp = () => {
+    state.currentTemp++;
+    changesTempDisplay()
+};
+
+const decreaseTemp = () => {
+    state.currentTemp--;
+    changesTempDisplay()
+};
```

Comment:
notice that these two methods do essentially the same thing but the difference is ++ or --.

How could you combine the logic of these two methods into one event handler called handleTempChange()?

What if you pass in the event object as an argument and you check if the event's target has an id `increase-t` or `decrease-t` and depending on the id then you could ++ or --

```javascript
const handleTempChange = (event) => {
    if (event.target.id === 'increase-t') {
        state.currentTemp++;
    } else {
        state.currentTemp--;
    }
    changesTempDisplay();
};